.github/workflows/test.yml

```
@@ -70,6 +69,23 @@ jobs:
         run: |
           git lfs checkout
 
+      - name: Fetch task spooler
+        uses: actions/checkout@v4
+        with:
+          repository: dsroberts/tsp_for_hpc
+          path: ./tsp
```

```suggestion
          path: ./tsp
          ref: v1.0.0
```
I've dropped a release for tsp, this way any further experimental changes don't break our CI